docs/user_guide/mac_install.md

```diff
@@ -120,33 +102,37 @@ Error: curses.h not found
    Open `rst/.profile/base.bash` to check paths are correctly set:
 
    `XPATH, NETCDF_PATH, CDF_PATH` 
-   To check if the paths are set correctly locate the following header files:
-   - For XPATH `locate png.h`
-   - For NETCDF_PATH `locate netcdf.h`
-   - For CDF_PATH `locate cdf.h`
-  
-   These locate commands will point to the `\include` sub directory for the respective library. Make sure your path ends in the parent directory above this (e.g., `/Applications/cdf/cdf39_0-dist` and not `/Applications/cdf/cdf39_0-dist/include`. 
+   To change the paths to the locations of appropriate header files:
+   - For XPATH, this is the location of X11, which is included with XCode. It should be `"opt/X11/"` for ARM-based Macs and `"usr/X11/"` for x86 Macs. The `/include` folder in here contains .h files (headers) that RST requires, like `png.h`.
+     
+   - For NETCDF_PATH, point this to where you installed netcdf. This should be `"/usr/local/Cellar/netcdf/X.X.X_X"` if you installed it with Homebrew for both types of Macs, where the X's are the version number.
+     
+   - For CDF_PATH, this will be where the CDF package is installed. It should be something like `"/Applications/cdf/cdf"`. The directory pointed to should contain the CDF `ReadMe.txt`.
    
    - If you have **IDL**, check to see that `IDL_IPATH` in `rst/.profile/idl.bash` is correct.
    	(Note: for users without IDL, modifying the `IDL_IPATH` environment variable is
    	not required).
 
-2. Load the RST environment variables. Open and edit your `~/.bashrc` file to include:
+3. When compiling RST later, `png.c`, located in `/RSTINSTALLLOCATION/codebase/base/src.lib/graphic/fbuffer.1.19/src/png.c`, doesn't seem to find the `png.h` header file correctly, even though it is located in the XPATH defined in the previous step. As a patch fix while we work out what to do more cleanly, you need to change a line to point directly to `png.h`
+
+	- Go to `png.c` and change line 32 `#png.h>` to `#include </opt/X11/include/png.h>` or `#include </usr/X11/include/png.h>`, for ARM and x86-based Macs, respectively. If your compilation still errors on `png.c`, make sure this is pointed to `png.h` correctly on your machine.
```

Comment:
This is too specific.  My png.h header was not located under X11.  Instead, I recommend directing users to find the location of their png.h header using something like: `find /opt/ -type f -name 'png.h'` or `find /usr/ -type f -name 'png.h'`